wordpress-mobile/WordPress-iOS
# Issue/show featured image imediately

Fixes #8997 

To test:
 - Start a new post
 - Insert some images on it
 - Tap on Async Publish
 - Check that one of the images of the post is show on the Post list page while the post is being uploaded.
